Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM quay.io/aperture147/ubuntu:focal
```

Comment:
@aperture147 Can we use container from docker hub instead of personal container ?